File: Maximum of 3 Numbers/max3_from_module.py
def max3(a, b, c):
    large = a

    if b > large:
        large = b
    if c > large:
        large = c

    return large

# example values:
a = 10
b = 3
c = 6

print("Large:", max3(a, b, c))  # prints 10

File: Maximum of 3 Numbers/max3_from_module.cpp
#include <iostream>

using namespace std;

int max3(int a, int b, int c) {
    int large = a;  // assuming the largest is a

    if (b > large) { large = b; }   // if b is larger than large, set large = b
    if (c > large) { large = c; }   // if c is larger than large, set large = c

    return large;
}

int main(void) {
    // example values:
    int a = 10, b = 3, c = 6;

    cout << "Large: " << max3(a, b, c) << endl;     // prints 10

    return 0;
}